docs/submission/submitting-molecular-data.md

```diff
@@ -300,13 +304,13 @@ Wrote manifest file 'manifest.txt' for analysisId 'a4142a01-1274-45b4-942a-01127
 The manifest will be written out to the score directory. You can optionally define a specific output directory for your manifest file using the `-f` parameter. If the directory does not yet exist, it will be created.
 
 ```shell
-./bin/sing manifest -a a4142a01-1274-45b4-942a-01127465b422 -f manifest.txt -f /my/output/dir/manifest.txt
+./bin/sing manifest -a a4142a01-1274-45b4-942a-01127465b422 -f manifest.txt -d /submitting/file/directory

```

Comment:
i would re write step 3 as

Use the returned `analysis_id` from step 2 to generate a manifest for file upload using the song-client `manifest` command. This manifest will be used with the score-client in the next step.

```shell
./bin/sing manifest -a a4142a01-1274-45b4-942a-01127465b422 -f /some/output/dir/manifest.txt  -d /submitting/file/directory

Wrote manifest file 'manifest.txt' for analysisId 'a4142a01-1274-45b4-942a-01127465b422'
```
The `manifest.txt` file will be written out to the directory ` /some/output/dir/`. If the output directory does not exist, it will be automatically created.

Reply:
@rtisma good sugggestions: how about this: 
![image](https://user-images.githubusercontent.com/15824252/84431873-06a6f480-abfa-11ea-8dfb-80db05a8051f.png)

Reply:
change the last bullet to

- the output file path for the manifest file with the `-f` parameter